bitnami/etcd/templates/cronjob.yaml

```diff
@@ -31,6 +31,8 @@ spec:
               command:
                 - /scripts/save-snapshot.sh
               env:
+              - name: BITNAMI_DEBUG
```

Comment:
I would add a different value to reflect specifically this functionality. I mean, something like `disasterRecovery.debug`.

Reply:
This is not for helm chart functionality, this is for debugging scripts and it is already there in the values: `images.debug: true`.
This conforms with the other uses such as in https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/statefulset.yaml#L97 and this is used for debugging bash scripts like in https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/scripts-configmap.yaml#L30 for etcd setup or https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/scripts-configmap.yaml#L193 pre-stop for etcd or probes https://github.com/bitnami/charts/blob/master/bitnami/etcd/templates/scripts-configmap.yaml#L245

So the cron job pod was missing this env var.
Another option would be to set:
```bash
: {BITNAMI_DEBUG:-"false"}
```
in every shell script but then still it would make it harder to use in the case of the cronjob (because it would always be set to false).

Reply:
Edited the comment to make it more easier to understand.

Reply:
Sorry, I didn't explain well, what I wanted to say here was that it would be good to have both features in different values. That would mean, `image.debug=true` if we want to debug the main container stuff, I mean,  statefulset containers, and `disasterRecovery.debug=true` if we want to debug this specific cronjob for the snapshots. @carrodher or @juan131 could you add your thought here, please?

Reply:
Ah. Well, in that case it's a feature and not a bug. :)